packages/web-components/src/components/content-block/content-block-heading.ts

```diff
@@ -37,6 +42,16 @@ class DDSContentBlockHeading extends StableSelectorMixin(LitElement) {
     super.connectedCallback();
   }
 
+  firstUpdated() {
+    this.content = stripHTML(this.innerHTML);
+    render(
+      html`
+        <h2>${this.content}</h2>
```

Comment:
@RaphaelAmadeu the additional heading markup is causing the actual render to look a bit off. If we just do the `stripHTML` and not add the `h2`, how does Bing renderer capture the lightDOM?